easybuild/easyconfigs/f/fastQValidator/fastQValidator-0.1.1a-20151214-goolf-1.7.20-aadc6f9.eb

```diff
@@ -0,0 +1,55 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'fastQValidator'
+version = '0.1.1a-20151214'
+fastqvalidator_git_commit = 'aadc6f9'
+libstatgen_git_commit = '8246906'
+versionsuffix = '-%s' % fastqvalidator_git_commit
+
+homepage = 'http://genome.sph.umich.edu/wiki/FastQValidator'
+description = """Validate FastQ Files"""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+# we download latest source available in github for fastQValidator and libStatGen (dependency)
+# using latest release tags doesn't work
+# https://github.com/statgen/fastQValidator/issues/9
+source_urls = [
+    'https://github.com/statgen/fastQValidator/archive/', 
+    'https://github.com/statgen/libStatGen/archive/'
+]
+
+sources = [
+    '%s.tar.gz' % fastqvalidator_git_commit, 
+    '%s.tar.gz' % libstatgen_git_commit
```

Comment:
actually, thinking about this again, it would be a pretty weird coincidence if you could download the same commit from another repo...

so, I'll go ahead and merge this as-is, thanks @pescobar!